doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -7,60 +7,68 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Lab Requirement](#4.2)
 * [4.3 Lab Topology](#4.3)
-* [4.4 Lab HW Spec](#4.4)
-* [4.5 Lab Use Guidelines](#4.5)
+* [4.4 Lab Use Guidelines](#4.4)
 
 
 <a name="4.1"></a>
-## 4.1 Introdcution
+## 4.1 Introduction
+This chapter provides the laboratory hardware requirements needed to deploy the reference implementation.  These requirements represent a minimum set of requirements, where different labs may have hardware or capabilities exceeding these minimum specifications.  
 
+The lab provides a set of physical servers, supported by a dedicated network infrastructure.  Users should be provided access to the servers, their operating systems, and out-of-band management (i.e. IPMI, IDRAC, ILO, etc.).  
 
 <a name="4.2"></a>
-## 4.2 Lab Requirement
-
-- Controller Nodes:
-  - 3 x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD.
-    - 3.2TB SSD.
-- Compute Node
-  - 10x
-    - 2x dual-port 25Gbps NIC.
-    - 2.2GHz 24C/48T.
-    - 512GB RAM.
-    - 2TB HDD
-    - Supports 2x Nodes for B/N/C flavors
-- Jump/Baremetal manager
-  - 1x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD
-- Networking
-  - 1x Spine Switch
-    - Total: 32x100G
-  - 2x Leaf Switch
-    - 48 x 25/10G
-    - 6 x 100G
-- 1x 48u Rack
-- Cables and Transcievers
-
-<a name="4.3"></a>
-## 4.3 Lab Topology
+## 4.2 Lab Requirements
 
+Labs are organized into one or more *Pods*, where the *Pod* provides the complete set of resources [servers and network(s)] necessary to support the reference implementation installation and operation.  
 
-<a name="4.4"></a>
-## 4.4 Lab HW Spec
+A CNTT compliant *Pod* shall meet the following requirements.
+
+1. One (1) physical server dedicated as a Jump / Test Host
+2. Six (6) physical servers, serving as either compute or controllers
+3. A configured network topology allowing for: Out-of-Band Management, Admin, Public, Private, and Storage Networks
+
+The Jump Host / Test Host system will be utilized to install and manage the operation of the *Pod*.  For example, the system can be used to host the installer software used to deploy and configure OpenStack on the other physical servers.  Once installed, the system may be used to run test automations or it may be used to generate test traffic intended to measure the performance of a VNF operating within the *Pod*.  To support these operations, the physical server provided for the Jump / Test Host shall meet the same hardware requirements as compute / controller, as outline below.  In this context, the system may be considered as a miscellaneous use  or general purpose system.
+
+<a name="4.2.1"></a>
+### 4.2.1 Physical Server Requirements
+Each server shall meet the following minimum specifications:
+
+- **CPU**
+  - 2x x86_64 CPU sockets (both populated), providing 24 cores each, 48 simultaneous multi-threads (SMT), at 2.2 GHz
```

Comment:
This is the _minimum_ configuration, so it's Broadwell. If we make it Cascade, it will disqualify the only running labs we currently have, which are Broadwell-based.

Reply:
I think I am a bit confused. I thought this was to specify something that $FirstLabLocation would purchase, not to align with an existing hardware specification.